src/IO/stir_ecat_common.cxx

```diff
@@ -208,7 +208,7 @@ find_timing_poss_sequence(const ProjDataInfo& pdi)
   for (int timing_pos_num = 1; timing_pos_num <= max_timing_pos_num; ++timing_pos_num)
     {
       timing_pos_sequence[2 * timing_pos_num - 1] = timing_pos_num;
```

Comment:
this cannot be correct. Even the positive bins have a different number now. I think it has to be
```suggestion
      timing_pos_sequence[2 * timing_pos_num - 1] = max_timing_pos_num + timing_pos_num;
```
Note that comment line a few lines earlier will need to be adapted as well.